core/templates/dev/head/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.html

```diff
@@ -12,18 +12,36 @@
       <i class="material-icons">play_arrow</i>
     </button>
   </div>
-  <div class="oppia-content-wrapper oppia-audio-bar-info">
-    <div ng-if="!cannotRecord && !checkingMicrophonePermission">
-      No audio recorded.<span ng-if="canVoiceover"> Press <i class="material-icons">mic</i>  to start recording.</span>
-    </div>
-    <div ng-if="checkingMicrophonePermission">
-      Loading microphone.<loading-dots></loading-dots>
-    </div>
-    <div ng-if="recordingPermissionDenied && cannotRecord">
-      You need to grant permission for this application to use your microphone.
+  <div class="d-block d-xl-none oppia-content-wrapper oppia-audio-bar-info">
+    <div class="padding-small-screen">
+      <div ng-if="!cannotRecord && !checkingMicrophonePermission">
+        <span ng-if="canVoiceover"> Press <i class="material-icons">mic</i>  to start recording.</span>
+      </div>
+      <div ng-if="checkingMicrophonePermission">
+        Loading microphone.<loading-dots></loading-dots>
+      </div>
+      <div ng-if="recordingPermissionDenied && cannotRecord">
+        Grant permission to use your microphone.
```

Comment:
Was there a reason to change this message, since this one feels more like a command. Can you change it back?

Reply:
I had to shorten the text so that it fits the space. I am not sure what else to do.